internal/controller/types.go

```diff
@@ -97,6 +97,40 @@ type OLSConfig struct {
 	CertificateDirectory string `json:"certificate_directory,omitempty"`
 	// Enable Introspection features
 	IntrospectionEnabled bool `json:"introspection_enabled,omitempty"`
+	// LLM Token Quota Configuration
+	QuotaHandlersConfig QuotaHandlersConfig `json:"quota_handlers,omitempty"`
```

Comment:
It may be better to make `QuotaHandlersConfig` optional by setting it to a pointer, so that we dont generate this section in the configmap passed to the service app pod, less coupling to the database if  the quota is not set.